.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor/
 /bin/
-composer.lock
\ No newline at end of file
+composer.lock
+.idea/
```

Comment:
This line should not be a part of .gitignore. Instead of adding IDE folders to each repo its better to use [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)